packages/kubernetes-1.30/kubelet-config

```diff
@@ -161,6 +161,12 @@ containerLogMaxSize: {{settings.kubernetes.container-log-max-size}}
 {{#if settings.kubernetes.container-log-max-files includeZero=true}}
 containerLogMaxFiles: {{settings.kubernetes.container-log-max-files}}
 {{/if}}
+{{#if settings.kubernetes.container-log-max-workers includeZero=true}}
+containerLogMaxWorkers: {{settings.kubernetes.container-log-max-workers}}
+{{/if}}
```

Comment:
Are you certain you don't need guard these via `if_not_null` blocks? Have you tested this on a variant build that isn't using your new settings SDK with the newer `KubernetesSettings` shape?

Reply:
TIA, I haven't test such case. I will make this change you suggested and test that use case!